template/test-report.hbs

```diff
@@ -47,6 +54,21 @@
 
   <div class="layout-stacked__content">
     <div class="layout-container--short" style="padding-top: 35px;">   
+        <div class="filter-box">
```

Comment:
This should probably be excluded from printed output, I believe `snyk-to-html` is used to generate PDFs a fair amount

Reply:
Interesting... differing output for print is a new req though, and would demand a new param. Mixed feelings about doing that in the same PR